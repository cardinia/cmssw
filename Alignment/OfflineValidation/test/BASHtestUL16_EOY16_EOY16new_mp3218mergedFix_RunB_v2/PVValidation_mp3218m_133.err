27-Sep-2019 21:42:38 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/20000/8E39208E-9509-E911-B000-24BE05C62611.root
27-Sep-2019 21:42:42 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/20000/8E39208E-9509-E911-B000-24BE05C62611.root
Begin processing the 1st record. Run 275311, Event 481764115, LumiSection 381 on stream 0 at 27-Sep-2019 21:42:50.801 CEST
Begin processing the 1001st record. Run 275311, Event 495657797, LumiSection 392 on stream 0 at 27-Sep-2019 21:46:14.228 CEST
Begin processing the 2001st record. Run 275311, Event 499817514, LumiSection 396 on stream 0 at 27-Sep-2019 21:48:17.588 CEST
Begin processing the 3001st record. Run 275311, Event 505243746, LumiSection 400 on stream 0 at 27-Sep-2019 21:50:21.345 CEST
Begin processing the 4001st record. Run 275311, Event 510141152, LumiSection 404 on stream 0 at 27-Sep-2019 21:52:21.125 CEST
Begin processing the 5001st record. Run 275311, Event 515004822, LumiSection 408 on stream 0 at 27-Sep-2019 21:54:13.582 CEST
Begin processing the 6001st record. Run 275311, Event 520090413, LumiSection 412 on stream 0 at 27-Sep-2019 21:56:13.920 CEST
Begin processing the 7001st record. Run 275311, Event 525131724, LumiSection 416 on stream 0 at 27-Sep-2019 21:58:13.814 CEST
Begin processing the 8001st record. Run 275311, Event 529233883, LumiSection 420 on stream 0 at 27-Sep-2019 22:00:16.759 CEST
Begin processing the 9001st record. Run 275311, Event 534554250, LumiSection 424 on stream 0 at 27-Sep-2019 22:02:11.514 CEST
Begin processing the 10001st record. Run 275311, Event 539770985, LumiSection 428 on stream 0 at 27-Sep-2019 22:04:07.440 CEST
Begin processing the 11001st record. Run 275311, Event 544506656, LumiSection 432 on stream 0 at 27-Sep-2019 22:06:11.903 CEST
Begin processing the 12001st record. Run 275311, Event 548960886, LumiSection 436 on stream 0 at 27-Sep-2019 22:08:08.757 CEST
Begin processing the 13001st record. Run 275311, Event 554315874, LumiSection 440 on stream 0 at 27-Sep-2019 22:10:09.526 CEST
Begin processing the 14001st record. Run 275311, Event 558427847, LumiSection 444 on stream 0 at 27-Sep-2019 22:12:09.590 CEST
Begin processing the 15001st record. Run 275311, Event 564182502, LumiSection 449 on stream 0 at 27-Sep-2019 22:14:05.687 CEST
Begin processing the 16001st record. Run 275311, Event 569792424, LumiSection 453 on stream 0 at 27-Sep-2019 22:16:04.995 CEST
Begin processing the 17001st record. Run 275311, Event 574378511, LumiSection 457 on stream 0 at 27-Sep-2019 22:17:58.898 CEST
Begin processing the 18001st record. Run 275311, Event 578677954, LumiSection 461 on stream 0 at 27-Sep-2019 22:20:04.370 CEST
Begin processing the 19001st record. Run 275311, Event 584058517, LumiSection 465 on stream 0 at 27-Sep-2019 22:22:16.587 CEST
Begin processing the 20001st record. Run 275311, Event 590247289, LumiSection 470 on stream 0 at 27-Sep-2019 22:24:18.916 CEST
Begin processing the 21001st record. Run 275311, Event 594970243, LumiSection 474 on stream 0 at 27-Sep-2019 22:26:25.967 CEST
Begin processing the 22001st record. Run 275311, Event 599019070, LumiSection 478 on stream 0 at 27-Sep-2019 22:28:34.589 CEST
Begin processing the 23001st record. Run 275311, Event 604885898, LumiSection 482 on stream 0 at 27-Sep-2019 22:30:37.648 CEST
Begin processing the 24001st record. Run 275311, Event 609160016, LumiSection 486 on stream 0 at 27-Sep-2019 22:32:42.661 CEST
Begin processing the 25001st record. Run 275311, Event 613591811, LumiSection 490 on stream 0 at 27-Sep-2019 22:34:54.483 CEST
Begin processing the 26001st record. Run 275311, Event 618206154, LumiSection 494 on stream 0 at 27-Sep-2019 22:36:59.052 CEST
Begin processing the 27001st record. Run 275311, Event 625265810, LumiSection 499 on stream 0 at 27-Sep-2019 22:38:58.708 CEST
Begin processing the 28001st record. Run 275311, Event 628983462, LumiSection 503 on stream 0 at 27-Sep-2019 22:41:14.529 CEST
Begin processing the 29001st record. Run 275311, Event 634595942, LumiSection 507 on stream 0 at 27-Sep-2019 22:43:18.350 CEST
Begin processing the 30001st record. Run 275311, Event 641175039, LumiSection 513 on stream 0 at 27-Sep-2019 22:45:31.075 CEST
Begin processing the 31001st record. Run 275311, Event 645827571, LumiSection 517 on stream 0 at 27-Sep-2019 22:47:41.816 CEST
Begin processing the 32001st record. Run 275311, Event 651216732, LumiSection 521 on stream 0 at 27-Sep-2019 22:49:52.393 CEST
Begin processing the 33001st record. Run 275311, Event 656011742, LumiSection 525 on stream 0 at 27-Sep-2019 22:51:53.663 CEST
Begin processing the 34001st record. Run 275376, Event 2035779521, LumiSection 1284 on stream 0 at 27-Sep-2019 22:52:16.050 CEST
Begin processing the 35001st record. Run 275376, Event 2044176932, LumiSection 1290 on stream 0 at 27-Sep-2019 22:52:17.362 CEST
Begin processing the 36001st record. Run 275376, Event 2051611693, LumiSection 1295 on stream 0 at 27-Sep-2019 22:52:18.891 CEST
Begin processing the 37001st record. Run 275376, Event 2059534557, LumiSection 1301 on stream 0 at 27-Sep-2019 22:52:20.071 CEST
Begin processing the 38001st record. Run 275376, Event 2066673387, LumiSection 1306 on stream 0 at 27-Sep-2019 22:52:21.420 CEST
Begin processing the 39001st record. Run 275376, Event 2074676194, LumiSection 1312 on stream 0 at 27-Sep-2019 22:52:22.498 CEST
Begin processing the 40001st record. Run 275376, Event 2082486454, LumiSection 1317 on stream 0 at 27-Sep-2019 22:52:23.628 CEST
Begin processing the 41001st record. Run 275376, Event 2089961680, LumiSection 1323 on stream 0 at 27-Sep-2019 22:52:24.843 CEST
Begin processing the 42001st record. Run 275376, Event 2098165403, LumiSection 1329 on stream 0 at 27-Sep-2019 22:52:25.954 CEST
Begin processing the 43001st record. Run 275376, Event 2106265125, LumiSection 1335 on stream 0 at 27-Sep-2019 22:52:27.091 CEST
Begin processing the 44001st record. Run 275376, Event 2112902043, LumiSection 1340 on stream 0 at 27-Sep-2019 22:52:28.267 CEST
Begin processing the 45001st record. Run 275376, Event 2122267789, LumiSection 1346 on stream 0 at 27-Sep-2019 22:52:29.737 CEST
Begin processing the 46001st record. Run 275376, Event 2129137900, LumiSection 1351 on stream 0 at 27-Sep-2019 22:52:31.117 CEST
Begin processing the 47001st record. Run 275376, Event 2137370755, LumiSection 1357 on stream 0 at 27-Sep-2019 22:52:32.339 CEST
Begin processing the 48001st record. Run 275376, Event 2145636011, LumiSection 1363 on stream 0 at 27-Sep-2019 22:52:33.828 CEST
Begin processing the 49001st record. Run 275376, Event 2152497687, LumiSection 1368 on stream 0 at 27-Sep-2019 22:52:35.154 CEST
27-Sep-2019 22:52:36 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/20000/8E39208E-9509-E911-B000-24BE05C62611.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[9.141MB/9.141MB][100%][==================================================][9.141MB/s]  [9.141MB/9.141MB][100%][==================================================][9.141MB/s]  
