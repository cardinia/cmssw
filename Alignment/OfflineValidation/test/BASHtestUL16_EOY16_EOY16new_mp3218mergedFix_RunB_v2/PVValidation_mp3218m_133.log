000 (3196549.133.000) 09/27 21:31:59 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196549.133.000) 09/27 21:42:15 Started transferring input files
	Transferring to host: <188.184.205.113:9618?addrs=188.184.205.113-9618+[2001-1458-301-7d--100-16d]-9618&noUDP&sock=4178_8763_14048>
...
040 (3196549.133.000) 09/27 21:42:15 Finished transferring input files
...
001 (3196549.133.000) 09/27 21:42:15 Job executing on host: <188.184.205.113:9618?addrs=188.184.205.113-9618+[2001-1458-301-7d--100-16d]-9618&noUDP&sock=2320_e7cf_3>
...
006 (3196549.133.000) 09/27 21:42:23 Image size of job updated: 40040
	25  -  MemoryUsage of job (MB)
	25344  -  ResidentSetSize of job (KB)
...
006 (3196549.133.000) 09/27 21:47:23 Image size of job updated: 552936
	537  -  MemoryUsage of job (MB)
	549544  -  ResidentSetSize of job (KB)
...
006 (3196549.133.000) 09/27 22:52:57 Image size of job updated: 1391904
	537  -  MemoryUsage of job (MB)
	549544  -  ResidentSetSize of job (KB)
...
040 (3196549.133.000) 09/27 22:52:57 Started transferring output files
...
040 (3196549.133.000) 09/27 22:52:57 Finished transferring output files
...
005 (3196549.133.000) 09/27 22:52:57 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:08:57, Sys 0 00:00:08  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:08:57, Sys 0 00:00:08  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	9051  -  Run Bytes Sent By Job
	1021  -  Run Bytes Received By Job
	9051  -  Total Bytes Sent By Job
	1021  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9408        1   2189377 
	   Memory (MB)          :      537     2000      2000 
...
