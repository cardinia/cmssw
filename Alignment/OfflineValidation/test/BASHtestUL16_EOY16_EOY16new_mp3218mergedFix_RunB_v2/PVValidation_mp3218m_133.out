Job started at  Fri Sep 27 21:42:15 CEST 2019
batch dir: /pool/condor/dir_10409 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_275311_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.2M -rw-r--r--. 1 tkello zh 9.2M Sep 27 22:52 PVValidation_mp3218m_275311.root -rw-r--r--. 1 tkello zh 21K Sep 27 21:42 PVValidation_mp3218m_275311_cfg.py -rw-r--r--. 1 tkello zh 7.7K Sep 27 22:52 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:42 _condor_stdout -rwxr-xr-x. 1 tkello zh 1021 Sep 27 21:42 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:42 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:42 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:42 var
Job ended at  Fri Sep 27 22:52:57 CEST 2019
